css/style.css

```diff
@@ -226,4 +226,71 @@ html {
 .btn--outline:active {
   background-color: rgba(0, 0, 0, 0.6);
   color: #f5f5f5;
+}
+
+/******************************/
+/* ABOUT SECTION */
+/******************************/
+
+.about-section {
+  background: url('../gallery/banners/about-banner.png');
+  background-size: cover;
+  padding-top: 7.6rem;
+  padding-left: 29rem;
+  width: 160rem;
+  height: 71rem;
+  display: inline-flex;
+  justify-content: flex-start;
+  box-shadow: inset 0 0 6rem rgb(0, 0, 0, 0.1);
+  position: relative;
+}
+
+.about-section:before {
+  content: '';
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to top left, rgba(85, 85, 85, 0.2) -100%, transparent 70%);
+}
+
+.our-mission {
+  position: relative;
+  width: 42rem;
+  padding-top: 2.2rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  align-items: center;
+  text-align: center;
+  margin-right: 6.4rem;
+}
+
+.slogan-our-mission {
+  font-family: "Yeseva One", sans-serif;
+  font-size: 3.6rem;
+  font-weight: 400;
+  color: #cc9900;
+}
+
+.decoration-about {
+  width: 24.8rem;
+  margin-bottom: 1.2rem;
+}
+
+.description-our-mission {
+  font-family: "Lato", sans-serif;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: #4c4c4c;
+  line-height: 2.4rem;
+  margin-bottom: 3.6rem;
+}
+
+.meal-img {
+  position: absolute;
+  height: 55.2rem;
```

Comment:
Setting the width is also important. BTW image size is `551px` x `551px`, not `552`